ledger-api/grpc-definitions/com/daml/ledger/api/v1/transaction_service.proto

```diff
@@ -39,6 +40,12 @@ service TransactionService {
   // Lookup a transaction by its ID.
   rpc GetFlatTransactionById (GetTransactionByIdRequest) returns (GetFlatTransactionResponse);
 
+  // Get the create and the consuming exercise event for the contract with the provided ID
```

Comment:
Why only consuming? Especially in scenarios where the contract was divulged to me, but I didn't witness the create, it would be useful to see the events that did the divulging.